WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderPaymentSection.swift

```diff
@@ -201,6 +213,8 @@ private extension OrderPaymentSection {
                         .footnoteStyle()
                         .multilineTextAlignment(.trailing)
                         .frame(width: nil, alignment: .trailing)
+                        .redacted(reason: self.viewModel.isLoading ? .placeholder : [])
+                        .shimmering(active: self.viewModel.isLoading)
```

Comment:
What do you think if we rename the bits that have both references to `viewModel` and `self.viewModel` within the same block? So it's easier to differentiate which one is controlling the properties when referencing the one inside or outside the closure. Example:
```diff
    @ViewBuilder var taxLines: some View {
-        ForEach(viewModel.taxLineViewModels, id: \.title) { viewModel in
+        ForEach(viewModel.taxLineViewModels, id: \.title) { taxLineViewModel in
            HStack {
                AdaptiveStack(horizontalAlignment: .leading, spacing: Constants.taxesAdaptativeStacksSpacing) {
-                    Text(viewModel.title)
+                    Text(taxLineViewModel.title)
                        .font(.footnote)
                        .fontWeight(.semibold)
                        .foregroundColor(Color(uiColor: .secondaryLabel))
                        .multilineTextAlignment(.leading)
                        .frame(maxWidth: .infinity, alignment: .leading)
-                    Text(viewModel.value)
+                    Text(taxLineViewModel.value)
                        .footnoteStyle()
                        .multilineTextAlignment(.trailing)
                        .frame(width: nil, alignment: .trailing)
-                        .redacted(reason: self.viewModel.isLoading ? .placeholder : [])
-                        .shimmering(active: self.viewModel.isLoading)
+                        .redacted(reason: viewModel.isLoading ? .placeholder : [])
+                        .shimmering(active: viewModel.isLoading)
                }
            }
        }
    }
```

Reply:
That's an excellent point 👍 I'll rename it.